addons/radar/script_component.hpp

```diff
@@ -0,0 +1,54 @@
+#define COMPONENT radar
+#include "\z\diwako_dui\addons\main\script_mod.hpp"
+
+// #define DEBUG_MODE_FULL
+// #define DISABLE_COMPILE_CACHE
+// #define CBA_DEBUG_SYNCHRONOUS
+// #define ENABLE_PERFORMANCE_COUNTERS
```

Comment:
macros that don't exist or aren't used anywhere in code.
Besides debug mode and compile cache the others shouldn't be here.